.github/workflows/automerge.yml

```diff
@@ -0,0 +1,41 @@
+name: Automerge
+# This is a function to automerge master and branch
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  # State github repository path
+  MY_REPO: https://GeospatialResearch:${{secrets.GITHUB_TOKEN}}@github.com/GeospatialResearch/Digital-Twins.git
+
+  # State master
+  LONG_LIVED_BRANCH: master
+
+  # State path to master repository path
+  MASTER_REPO: https://github.com/GeospatialResearch/Digital-Twins.git
+
+  # State branch
+  DEFAULT_BRANCH: otakaro_digital_twin
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.PUSH_PAGES_PAT }}
```

Comment:
You will need to specify a branch here (DEFAULT_BRANCH) using `ref:` https://github.com/actions/checkout